Makefile

```diff
@@ -22,7 +22,7 @@ VERSION_VARIABLES = DdevVersion WebImg WebTag DBImg DBTag RouterImage RouterTag
 # These variables will be used as the default unless overridden by the make
 DdevVersion ?= $(VERSION)
 WebImg ?= drud/nginx-php-fpm-local
-WebTag ?= v0.8.2
+WebTag ?= 20171204_version_simplification
```

Comment:
Is this the tag format that we're moving to? Or a temporary thing?

Reply:
This is a temporary tag so that the PR can be tested against the updated web image. Once we complete a container image-related task, we tag a new release for that container, and this tag gets updated to the new release tag for that container before it gets merged in.

Reply:
Got it. That makes sense.